### PR TITLE
genus uniquification fix

### PR DESCRIPTION
**Pull request by jjthomas**

### CODE REVIEW

**Comment by alexcarsello** [comment]
Can we run a test to see if this fixes the LVS error before merging?

**Review by alexcarsello — APPROVED** [review]
Approving as long as it fixes LVS issue.

**Comment by jjthomas** [comment]
LVS is correct for tile array, merging.

**Pull request merged**
